From: Heiko Stuebner <heiko@sntech.de>
Subject: [PATCH] Enable out of tree build

This also includes a backported gen-tags.sh to fix
build-failures.

Signed-off-by: Heiko Stuebner <heiko@sntech.de>

---
 lib/Makefile.am | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/lib/Makefile.am b/lib/Makefile.am
index 4a1e8d9..e52e76b 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -47,15 +47,19 @@ CLEANFILES = \
 # Hack to avoid using ylwrap. It does not function correctly in combination
 # with --header-file=
 route/pktloc_grammar.c: route/pktloc_grammar.l
+	$(AM_V_GEN) install -d route/
 	$(AM_V_GEN) $(FLEX) --header-file=route/pktloc_grammar.h $(LFLAGS) -o $@ $^
 
 route/pktloc_syntax.c: route/pktloc_syntax.y
+	$(AM_V_GEN) install -d route/
 	$(AM_V_GEN) $(YACC) -d $(YFLAGS) -o $@ $^
 
 route/cls/ematch_grammar.c: route/cls/ematch_grammar.l
+	$(AM_V_GEN) install -d route/cls/
 	$(AM_V_GEN) $(FLEX) --header-file=route/cls/ematch_grammar.h $(LFLAGS) -o $@ $^
 
 route/cls/ematch_syntax.c: route/cls/ematch_syntax.y
+	$(AM_V_GEN) install -d route/cls/
 	$(AM_V_GEN) $(YACC) -d $(YFLAGS) -o $@ $^
 
 libnl_route_3_la_LIBADD  = libnl-3.la
-- 
tg: (78abb19..) debian/out-of-tree (depends on: upstream-dist)
